### PR TITLE
Fix broken theme demo on Hugo Themes website

### LINKED ISSUE #241: Theme demo broken on the Hugo Themes website
Hello @olOwOlo 

I am one of the maintainers of Hugo Themes and I noticed that in the latest deploy your theme's demo failed to generate.

Also I noticed that [recently you migrated](https://github.com/olOwOlo/hugo-theme-even/commit/5657112dfb0e6eac2faa2ae1dc0978ed29030709) to using Hugo Pipes.

However you need to take additional steps to make the demo generate on the Hugo Themes Showcase. See here for detailed instructions: https://github.com/gohugoio/hugoThemes#resources

If you have any questions feel free to ask, otherwise feel free to close this issue once you implement the necessary changes.

### PR DESCRIPTION
**Pull request by panda-z**
Resolves #241

### CODE REVIEW

**Comment by olOwOlo** [comment]
Thank you for your contribution.

**Comment by panda-z** [comment]
You're welcome :D

File: README-zh.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ $ git clone https://github.com/olOwOlo/hugo-theme-even themes/even
 
 **重要:** 在主题的 [`exampleSite`](https://github.com/olOwOlo/hugo-theme-even/tree/master/exampleSite) 目录下有一个 [`config.toml`](https://github.com/olOwOlo/hugo-theme-even/blob/master/exampleSite/config.toml) 文件，**将这个 [`config.toml`](https://github.com/olOwOlo/hugo-theme-even/blob/master/exampleSite/config.toml) 文件复制到你的站点目录下**，根据自己的需求更改即可。
 
+**重要:** 本主题用到了 [Hugo Pipes](https://gohugo.io/hugo-pipes/introduction/) 功能。如需修改 `asserts` 目录下的文件，请安装 extended 版。
```
Comment (panda-z) [inline README-zh.md:26]: @olOwOlo Please correct the typo here. `asserts` should be `assets`. The English README should also be corrected.

Comment (olOwOlo) [reply README-zh.md:26]: Fixed.

**Pull request merged**